src/main/java/calculator/Delimiter.java

```diff
@@ -0,0 +1,35 @@
+package calculator;
+
+
+import java.util.Arrays;
+import java.util.regex.Pattern;
+import java.util.stream.Stream;
+
+public class Delimiter {
+
+    public static final String CUSTOM_DELIMITER_PREFIX = "//";
+    public static final String CUSTOM_DELIMITER_SUFFIX = "\n";
+
+    private static final String DEFAULT_DELIMITER = "[,:]";
+    private final String delimiter;
```

Comment:
상수와 인스턴스 변수는 줄바꿈을 통해서 구분해주면 가독성을 높일 수 있을 것 같아요. 🙂
```suggestion
    private static final String DEFAULT_DELIMITER = "[,:]";
    
    private final String delimiter;
```